modules/seqexec/web/shared/src/main/scala/seqexec/web/common/Zipper.scala

```diff
@@ -3,57 +3,87 @@
 
 package seqexec.web.common
 
+import cats._
 import cats.implicits._
 import cats.data.NonEmptyList
-import cats._
+import monocle.macros.Lenses
+import monocle.{ Prism, Traversal }
 
 /**
- * Minimal zipper based on scalaz's implementation
- * This is only meant for small collections. performance has not been optimized
- */
+  * Minimal zipper based on scalaz's implementation
+  * This is only meant for small collections. performance has not been optimized
+  */
+@Lenses
 final case class Zipper[A](lefts: List[A], focus: A, rights: List[A]) {
+
```

Comment:
We should be able to use `cats.collection.Zipper` pretty soon.

Reply:
ok, this is fairly simple so we may not need more for this use case